tests/test_lib_utils.py

```diff
@@ -215,9 +216,14 @@ def test_07_parse_date(self):
         d = parse_date("23.12.2016 6:30")
         self.assertEqual(d, datetime(2016, 12, 23, hour=6, minute=30))
 
-        # Non-matching date returns None
         d = parse_date("23.12.16")
-        self.assertEqual(d, None)
+        self.assertEqual(d, datetime(2016, 12, 23, 0, 0))
```

Comment:
We should probably add a test for the MM/DD confusion here. Right now, ``parse_date`` behaves incorrectly (see above):
<pre>
>>> privacyidea.lib.utils.parse_date('03.04.16')
datetime.datetime(2016, 3, 4, 0, 0)
</pre>